tools/update-components.sh

```diff
@@ -13,107 +13,107 @@ TFLITE_REPO_URL="https://github.com/espressif/tflite-micro-esp-examples.git"
 #
 # CLONE/UPDATE ESP32-CAMERA
 #
-echo "Updating ESP32 Camera..."
-if [ ! -d "$AR_COMPS/esp32-camera" ]; then
-	git clone $CAMERA_REPO_URL "$AR_COMPS/esp32-camera"
-else
-	git -C "$AR_COMPS/esp32-camera" fetch && \
-	git -C "$AR_COMPS/esp32-camera" pull --ff-only
-fi
-if [ $? -ne 0 ]; then exit 1; fi
+#echo "Updating ESP32 Camera..."
```

Comment:
we can actually leave esp32-camera here, because it is not required to build Arduino as component. We do not expand on it's API on our side. Much like TFLite

Reply:
Camera is now in the yml file. It is installed automatically by the PACMAN.